tests/meson.build

```diff
@@ -160,6 +160,18 @@ if gtest_dep.found()
           install_dir: unittest_install_dir)
 
       test('unittest_mqtt', unittest_mqtt, env: testenv)
+
+      # ntp util test
+      gmock_dep = dependency('gmock', required: false)
+      if get_option('enable-tizen') and gmock_dep.found()
```

Comment:
Nice work!
Ubuntu 18.04 does not support `libgmock-dev` package so this condition looks OK!

Reply:
Right. I excluded it from Ubuntu because we distribute packages on both 18.04 and 20.04 from Launchpad.